sdk/storage/Azure.Storage.Files.DataLake/tests/SessionRecords/FileClientTests/DownloadToAsync_PathOverloads - Copy.json

```diff
@@ -0,0 +1,287 @@
+{
```

Comment:
Do we want all these tests with `- Copy` suffixes?

Reply:
deleted.